web/client/components/map/openlayers/plugins/WMSLayer.js

```diff
@@ -170,7 +206,38 @@ const createLayer = (options, map) => {
             resolutions: options.resolutions || MapUtils.getResolutions(),
             tileSize: options.tileSize ? options.tileSize : 256,
             origin: options.origin ? options.origin : [extent[0], extent[1]]
-        })
+        }),
+        tileLoadFunction: (tile, src) => {
+            if (typeof window.btoa === 'function' && src.length >= (options.maxLengthUrl || Infinity)) {
```

Comment:
What are the differences in imageLoadFunction for single tile and tileLoadFunction for tiles? Is there any chance we can have a simpler function for both?

Reply:
i will see if we can unify them in a single load function
i'm sure we care simplify more, any idea apart from abstracting it?